src/app/containers/ErrorMain/index.jsx

```diff
@@ -1,18 +1,67 @@
 import React from 'react';
 import { number } from 'prop-types';
+import nanoid from 'nanoid';
+import styled from 'styled-components';
 import { Headline } from '@bbc/psammead-headings';
+import InlineLink from '@bbc/psammead-inline-link';
 import Paragraph from '@bbc/psammead-paragraph';
+import { C_POSTBOX } from '@bbc/psammead-styles/colours';
+import { GEL_PARAGON } from '@bbc/gel-foundations/typography';
+import { FF_NEWS_SANS_REG } from '@bbc/psammead-styles/fonts';
 import { GhostWrapper, GridItemConstrained } from '../../lib/styledGrid';
+import { ServiceContextConsumer } from '../../contexts/ServiceContext';
+
+const StatusCode = styled.span`
+  ${GEL_PARAGON}
+  color: ${C_POSTBOX};
+  display: block;
+  font-family: ${FF_NEWS_SANS_REG};
+  font-weight: 600;
+  padding-bottom: 0.5rem;
+`;
+
+const ShortHeadline = styled(Headline)`
+  padding: 2.5rem 0 2.5rem 0;
+`;
+
+const LongGridItemConstrained = styled(GridItemConstrained)`
+  padding-bottom: 4rem;
+`;
+
+const ListItem = Paragraph.withComponent('li');
```

Comment:
This could use `styled-component` V4's [polymorphic `as` prop](https://medium.com/styled-components/announcing-styled-components-v4-better-faster-stronger-3fe1aba1a112#84b4) in the map, & might be neater.

Reply:
Done :)